Sources/Views/MusicProgressSliderCell.m

```diff
@@ -0,0 +1,55 @@
+//
+//  MusicProgressSlider.m
+//  Hermes
+//
+//  Created by Xinhong LIU on 19/4/15.
+//
+//
+
+#import "MusicProgressSliderCell.h"
+
+@implementation MusicProgressSliderCell
+
+@synthesize knobImage = _knobImage;
```

Comment:
This line will become unnecessary with my other suggestions implemented, as will the `knobImage` and `setKnobImage:` methods.